### PR TITLE
Fix: AutoDJ causing freeze

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
Auto DJ would cause freezing issue of presence due to null value. Value now checked before updating to avoid this issue.

### CODE REVIEW

File: TruckersFM/dist/metadata.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "en": "The number one hit music station for Trucking Simulators. Radio made for driving, now playing online, in cab and on the go."
    },
   "url": "truckers.fm",
-  "version": "1.0",
+  "version": "1.1",
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json:11]: version 1.1.0

**Comment by Bas950** [comment]
and update your fork

**Comment by NinjaLabs-Dev** [comment]
Should be updated 😄

**Comment by NinjaLabs-Dev** [comment]
DeepScan fixed 👍

File: websites/T/TruckersFM/presence.ts [hunk 2]
```diff
@@ -7,7 +7,8 @@ presence.on("UpdateData", () => {
     document.querySelector(".player-artist-text").textContent
   } - ${document.querySelector(".player-title-text").textContent} `;
   const dj = document.querySelector(".live-name").textContent;
-  const liveTill = document.querySelector(".live-time").textContent;
+  let liveTill = null;
+  liveTill = document.querySelector(".live-time").textContent;
   const pageName = document.title.slice(13);
   const liveTime = liveTill.slice(6);
   const presenceData: PresenceData = {
```
[the remark this answers is not in the document]
Comment (doomlerd) [reply websites/T/TruckersFM/presence.ts:14]: [ref](https://eslint.org/docs/rules/one-var)

**Comment by doomlerd** [comment]
Update your fork again please.

**Comment by NinjaLabs-Dev** [comment]
Evidence of implementation and working https://ninjalabs.co.uk/i/3jm6oz1.png https://ninjalabs.co.uk/i/be6yfaf.png

**Pull request merged**